SwifterDemoMac/AppDelegate.swift

```diff
@@ -35,7 +35,8 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }
 
     @objc func handleEvent(_ event: NSAppleEventDescriptor!, withReplyEvent: NSAppleEventDescriptor!) {
-        Swifter.handleOpenURL(URL(string: event.paramDescriptor(forKeyword: AEKeyword(keyDirectObject))!.stringValue!)!)
+        let callbackUrl = URL(string: "swifter://success")!
+        Swifter.handleOpenUrl(URL(string: event.paramDescriptor(forKeyword: AEKeyword(keyDirectObject))!.stringValue!)!, callbackUrl: callbackUrl)
```

Comment:
Guard against fetching the parameters instead of force-unwrap.

Reply:
Fixed!
https://github.com/mattdonnelly/Swifter/pull/271/commits/02c434f06760c5136208c849d3fd1fff436c0a2e